e2e/cp_test.go

```diff
@@ -1004,6 +1004,165 @@ func TestCopyMultipleFilesToS3Bucket(t *testing.T) {
 	}
 }
 
+// adresses the #360
+// cp parent/*/name.txt s3://bucket/newfolder
+func TestCopyMultipleFilesWithWildcardedDirectoryToS3Bucket(t *testing.T) {
+
```

Comment:
Empy lines could be removed.

Reply:
I've removed this, and other empty lines, thanks for warning.